--- sg3_utils-1.19/Makefile.orig	2005-12-27 05:47:53.000000000 +0100
+++ sg3_utils-1.19/Makefile	2005-12-28 19:42:43.165757048 +0100
@@ -1,10 +1,10 @@
 SHELL = /bin/sh
 
 PREFIX=/usr/local
-LIBDIR=$(DESTDIR)/$(PREFIX)/lib
-INSTDIR=$(DESTDIR)/$(PREFIX)/bin
-MANDIR=$(DESTDIR)/$(PREFIX)/share/man
-INCLUDEDIR=$(DESTDIR)/$(PREFIX)/include
+LIBDIR=$(PREFIX)/lib
+INSTDIR=$(PREFIX)/bin
+MANDIR=$(PREFIX)/share/man
+INCLUDEDIR=$(PREFIX)/include
 
 CC = gcc
 LD = gcc
@@ -187,22 +187,21 @@
 	libtool --mode=link $(LD) -o $@ $(LDFLAGS) $^
 
 install: $(EXECS)
-	install -d $(INSTDIR)
-	install -d $(LIBDIR)
-	libtool --mode=install install -c libsgutils.la $(LIBDIR)/libsgutils.la
-	libtool --finish $(LIBDIR)
+	install -d $(DESTDIR)$(INSTDIR)
+	install -d $(DESTDIR)$(LIBDIR)
+	libtool --mode=install install -c libsgutils.la $(DESTDIR)$(LIBDIR)/libsgutils.la
+	libtool --finish $(DESTDIR)$(LIBDIR)
 	for name in $^; \
 	 do libtool --mode=install install -m 755 \
-		$$name $(INSTDIR); \
+		$$name $(DESTDIR)$(INSTDIR); \
 	done
-	install -d $(MANDIR)/$(MAN_PREF)
+	install -d $(DESTDIR)$(MANDIR)/$(MAN_PREF)
 	for mp in $(MAN_PGS); \
-	 do install -m 644 $$mp $(MANDIR)/$(MAN_PREF); \
-	 gzip -9f $(MANDIR)/$(MAN_PREF)/$$mp; \
+	 do install -m 644 $$mp $(DESTDIR)$(MANDIR)/$(MAN_PREF); \
 	done
-	install -d $(INCLUDEDIR)/scsi
+	install -d $(DESTDIR)$(INCLUDEDIR)/scsi
 	for hdr in $(HEADERS); \
-	 do install -m 644 $$hdr $(INCLUDEDIR)/scsi ; \
+	 do install -m 644 $$hdr $(DESTDIR)$(INCLUDEDIR)/scsi ; \
 	done
 
 uninstall:
